src/collectors/prometheus_rabbitmq_core_metrics_collector.erl

```diff
@@ -117,50 +117,43 @@
         {2, erlang_net_ticktime_seconds, gauge, "Inter-node heartbeat interval in seconds", net_ticktime}
     ]},
 
-    % These metrics are captured in Erlang Distribution collector, built-into prometheus.erl
-    % https://github.com/deadtrickster/prometheus.erl/pull/92
-    % {node_node_metrics, [
-    %     {2, peer_node_bytes_in, counter, "Bytes received from peer node", recv_bytes},
-    %     {2, peer_node_bytes_out, counter, "Bytes sent to peer node", send_bytes}
-    % ]},
-
     {node_persister_metrics, [
-        {2, io_read, counter, "I/O read operations", io_read_count},
-        {2, io_read_bytes, gauge, "I/O bytes read", io_read_bytes},
-        {2, io_read_time_microseconds, gauge, "I/O read total time in microseconds", io_read_time},
-        {2, io_write, counter, "I/O write operations", io_write_count},
-        {2, io_write_bytes, gauge, "I/O bytes written", io_write_bytes},
-        {2, io_write_time_microseconds, gauge, "I/O write total time in microseconds", io_write_time},
-        {2, io_sync, counter, "I/O sync operations", io_sync_count},
-        {2, io_sync_time_microseconds, gauge, "I/O sync total time in microseconds", io_sync_time},
-        {2, io_seek, counter, "I/O seek operations", io_seek_count},
-        {2, io_seek_time_microseconds, gauge, "I/O seek total time in microseconds", io_seek_time},
-        {2, io_open_attempt, counter, "File open attempts", io_file_handle_open_attempt_count},
-        {2, io_open_attempt_time_microseconds, gauge, "File open attempts total time in microseconds", io_file_handle_open_attempt_time},
-        {2, io_reopen, counter, "Number of times files have been reopened", io_reopen_count},
-        {2, schema_db_ram_tx, counter, "Schema DB memory transactions", mnesia_ram_tx_count},
-        {2, schema_db_disk_tx, counter, "Schema DB disk transactions", mnesia_disk_tx_count},
-        {2, msg_store_read, counter, "Message store read operations", msg_store_read_count},
-        {2, msg_store_write, counter, "Message store write operations", msg_store_write_count},
-        {2, queue_index_read, counter, "Queue Index read operations", queue_index_read_count},
-        {2, queue_index_write, counter, "Queue Index write operations", queue_index_write_count},
-        {2, queue_index_journal_write, counter, "Queue Index Journal write operations", queue_index_journal_write_count}
+        {2, io_read_ops_total, counter, "Total number of I/O read operations", io_read_count},
+        {2, io_read_bytes_total, counter, "Total number of I/O bytes read", io_read_bytes},
+        {2, io_read_time_microseconds_total, counter, "Total I/O read time", io_read_time},
```

Comment:
This should be in seconds. As-is you've a mix of two units, and seconds are the standard that exporters should expose.

Reply:
They are stored as microseconds, so the unit is correct. Since these would ideally be summaries, I will take a rain check on the conversion to seconds and change the metric type at the same time.